### PR TITLE
PIA-438: Prepared return assistant for localization.

### PR DESCRIPTION
**Pull request by maciejtrybilo**
English copy can be reviewed here. The German strings are in English atm pending translation.

### CODE REVIEW

File: GiniVision/Assets/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -120,3 +120,49 @@
...
+"ginivision.digitalinvoice.screentitle" = "Digital Invoice";
+"ginivision.digitalinvoice.paybuttontitle.noinvoice" = "Pay";
+"ginivision.digitalinvoice.paybuttontitle" = "Pay %d/%d";
+"ginivision.digitalinvoice.paybuttontitle.accessibilitylabel" = "Pay %d out of %d";
+
+"ginivision.digitalinvoice.items" = "Items %d/%d";
+"ginivision.digitalinvoice.items.accessibilitylabel" = "Items %d out of %d";
+"ginivision.digitalinvoice.whatisthisbutton" = "What is this?";
+
+"ginivision.digitalinvoice.whatisthisactionsheet.title" = "Is it helpful?";
```
Comment (cursork) [inline GiniVision/Assets/de.lproj/Localizable.strings]: I see `ginivision.digitalinvoice.whatisthisactionsheet.title` in the title box when running the app. Nonetheless, 'Is it helpful?' doesn't seem right? Should it be 'What is this?' again, just like the button?

File: GiniVision/Assets/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -120,3 +120,49 @@
...
+"ginivision.digitalinvoice.whatisthisactionsheet.message" = "We extracted the detailed items from your invoice to help you calculate the amount to pay in case you want to return some goods or simply not fully-paying it.";
```
Comment (cursork) [inline GiniVision/Assets/de.lproj/Localizable.strings]: Sentence is fine, but I had to read it twice... Possibly something a bit more broken up?
"We extracted the detailed items from your invoice. If you would like to return some items or partially pay the invoice, you can deselect them and the amount to pay will be recalculated."

File: GiniVision/Assets/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -120,3 +120,49 @@
...
+"ginivision.digitalinvoice.whatisthisactionsheet.action.helpful" = "Yes, that's helpful";
+"ginivision.digitalinvoice.whatisthisactionsheet.action.nothelpful" = "No, I don't want to see it again";
```
Comment (cursork) [inline GiniVision/Assets/de.lproj/Localizable.strings]: I'm confused by the button text here. They're just on an informational popup? The user won't dismiss that forever?

Or is the 'I don't want to see it again' meant to be about not wanting to see the entire digital invoice page again?

File: GiniVision/Assets/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -120,3 +120,49 @@
...
+"ginivision.digitalinvoice.lineitem.quantity" = "Quantity: %d";
+"ginivision.digitalinvoice.lineitem.editbutton" = "Edit";
+
+"ginivision.digitalinvoice.checkmarkbutton.deselect.accessibilitylabel" = "Deselect item";
+"ginivision.digitalinvoice.checkmarkbutton.select.accessibilitylabel" = "Select item";
+
+"ginivision.digitalinvoice.deselectreasonactionsheet.message" = "Your reasons to return this item";
+"ginivision.digitalinvoice.deselectreasonactionsheet.action.cancel" = "Cancel";
+
+"ginivision.digitalinvoice.deselectreason.looksdifferent" = "Looks different than site image";
```
Comment (cursork) [inline GiniVision/Assets/de.lproj/Localizable.strings]: "Looks different to image on site"

File: GiniVision/Assets/de.lproj/Localizable.strings (hunk 3 above)
Comment (maciejtrybilo) [reply GiniVision/Assets/de.lproj/Localizable.strings]: It's the latter.

File: GiniVision/Assets/de.lproj/Localizable.strings (hunk 1 above)
Comment (maciejtrybilo) [reply GiniVision/Assets/de.lproj/Localizable.strings]: Fixed.

File: GiniVision/Assets/de.lproj/Localizable.strings (hunk 2 above)
Comment (maciejtrybilo) [reply GiniVision/Assets/de.lproj/Localizable.strings]: Updated.

File: GiniVision/Assets/de.lproj/Localizable.strings (hunk 4 above)
Comment (maciejtrybilo) [reply GiniVision/Assets/de.lproj/Localizable.strings]: Updated.

File: GiniVision/Assets/de.lproj/Localizable.strings [hunk 5]
```diff
@@ -120,3 +120,49 @@
...
+
+"ginivision.digitalinvoice.deselectreason.looksdifferent" = "Looks different to image on site";
+"ginivision.digitalinvoice.deselectreason.poorqualityorfaulty" = "Poor quality/faulty";
+"ginivision.digitalinvoice.deselectreason.doesnotfit" = "Doesn't fit properly";
+"ginivision.digitalinvoice.deselectreason.doesnotsuit" = "Doesn't suit me";
+"ginivision.digitalinvoice.deselectreason.wrongitem" = "Received wrong item";
+"ginivision.digitalinvoice.deselectreason.damaged" = "Parcel damaged";
+"ginivision.digitalinvoice.deselectreason.arrivedtoolate" = "Arrived too late";
+
+"ginivision.digitalinvoice.headermessage.primary" = "Scan successfull!";
```
Comment (cursork) [inline GiniVision/Assets/de.lproj/Localizable.strings]: Successful - single 'l'

Comment (maciejtrybilo) [reply GiniVision/Assets/de.lproj/Localizable.strings]: Fixed

**Pull request merged**